src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1095,7 +1097,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
                 flags |= SF_TRUSTED;
             }
 
-            if (! app_.config().VALIDATION_PRIV.isSet())
+            if (! app_.config().VALIDATION_PUB.size())
```

Comment:
I thought I removed that. This shouldn't be here (only `VALIDATION_SEED` should be) and at the very least, this should be a `boost::optional<PublicKey>`. Thanks.